guava-tests/test/com/google/common/math/QuantilesTest.java

```diff
@@ -701,6 +701,15 @@ public void testScale_index_computeInPlace_empty() {
     }
   }
 
+  public void testEmpty_indexes(){
+    int[] empty_indexes = {};//visible for testing
+    try {
+      Quantiles.ScaleAndIndexes intermidiate = Quantiles.scale(10).indexes(empty_indexes);
+      fail("Expected IllegalArgumentException");
```

Comment:
`computeInPlace` is not called here, so I expect the bug is not triggered by this test case. Did you try to run the tests with your fix?

Reply:
Yes, this test case is intended not to fail. I want to test if it works fine with the fix.
Since `checkArgument()` is expected to throw `IllegalArgumentException` if `indexes` is empty.

Reply:
Ok, I understand.